draft-oid4vc-haip-sd-jwt-vc.md

```diff
@@ -139,7 +139,53 @@ Note: Issuers should be mindful of how long the usage of the refresh token is al
 
 ### Wallet Attestation Schema {#wallet-attestation-schema}
 
-[Section 3.1 of wallet attestation draft would define the basics, and this profile will define the details.]
+Wallets MUST use attestations following the definition given in [@!I-D.ietf-oauth-attestation-based-client-auth].
+
+In addition to this definition, the Wallet Attestation MAY contain the following claims in the `cnf` element:
+
+* `key_type`: OPTIONAL. JSON String that asserts the security mechanism the wallet uses to manage the private key associated with the public key given in the `cnf` claim. This mechanism is based on the capabilities of the execution environent of the wallet, this might be a secure element (in case of a wallet residing on a smartphone) or a Cloud-HSM (in case of a cloud wallet). This specification defines the following values for `key_type`:
+  * `software`: It MUST be used when the wallet uses software-based key management.
+  * `hardware`: It MUST be used when the wallet uses software-based key management.
+  * `tee`: It SHOULD be used when the wallet uses the Trusted Execution Environment for key management.
+  * `secure_enclave`: It SHOULD be used when the Wallet uses the Secure Enclave for key management.
+  * `strong_box`: It SHOULD be used when the Wallet uses the Strongbox for key management.
+  * `secure_element`: It SHOULD be used when the Wallet uses the Trusted Execution Environment for key management.
```

Comment:
definition about Secure Enclave, Strongbox and TEE are required if any external reference is given to the reader

what they are or where they are defined is an editorial requirement